### DISCUSSION TITLE
Where can I find the API Documentation for Collection Unlisted API?

### DISCUSSION THREAD

**Discussion by aks30** [Sunbird-Knowlg/Community#116] [Q&A] [answered]
I am looking for API documentation for the Collection unlisted API which is mentioned as an alternative to the now deprecated API of Content unlisted action here - (https://knowlg.sunbird.org/use/deprecations/release-5.2.0#apis)

I have tried looking at the postman collection links mentioned here - (https://knowlg.sunbird.org/learn/product-and-developer-guide/content-service-1/content-api) but couldn't the API - `{{host}}/api/collection/v1/unlisted/publish/{{collection_id}}`

For reference and context - https://github.com/orgs/Sunbird-Ed/discussions/398

Cc - @vijiurs @kiranharidas187 @Preethi-Sl @aishwaryashikshalokam

**Comment by pallakartheekreddy** [comment]
@aks30 We are working on API documentation for this. We will update this thread once we update the API documentation.

**Reply by aks30** [reply]
Sure @pallakartheekreddy , meanwhile what would help is if you can share the same out of your setup here. It will help us unblock and proceed on another issue we are checking with Ed BB. Refer - https://github.com/orgs/Sunbird-Ed/discussions/398

**Reply by pallakartheekreddy** [reply, marked as answer]
@aks30 Please use below CURL for unlisted publish API

```
curl --location '{{host}}/api/collection/v1/unlisted/publish/do_id' \
--header 'Content-Type: application/json' \
--header 'user-id: sunbird' \
--header 'Authorization: Bearer {{api_key}}' \
--data '{
    "request": {
        "content": {
            "lastPublishedBy": "user_id"
        }
    }
}'
```
